modules/caddyhttp/reverseproxy/command.go

```diff
@@ -175,6 +177,13 @@ func cmdReverseProxy(fs caddycmd.Flags) (int, error) {
 	if fromAddr.Scheme == "http" {
 		server.AutoHTTPS = &caddyhttp.AutoHTTPSConfig{Disabled: true}
 	}
+	if disableRedir {
+		if server.AutoHTTPS != nil {
+			server.AutoHTTPS = &caddyhttp.AutoHTTPSConfig{DisabledRidir: true}
+		} else {
+			server.AutoHTTPS.DisableRedir = true
+		}
```

Comment:
There was a typo, but also it can be simplified a bit:

```suggestion
		if server.AutoHTTPS != nil {
			server.AutoHTTPS = &caddyhttp.AutoHTTPSConfig{}
		}
		server.AutoHTTPS.DisableRedir = true
```